app/enterprise/0.31-x/getting-started/adding-your-api.md

```diff
@@ -93,7 +93,7 @@ Now that you've added your API to Kong EE, let's learn how to enable plugins.
 
 Go to [Enabling Plugins &rsaquo;][enabling-plugins]
 
-[API]: /latest/admin-api
+[API]: /0.12.x/admin-api
 [enabling-plugins]: /enterprise/{{page.kong_version}}/getting-started/enabling-plugins
-[proxy-port]: /latest/configuration/#nginx-section
+[proxy-port]: /0.12.x/configuration/#nginx-section
 [mockbin]: https://mockbin.com/
```

Comment:
This section doesn't appear on the autogenerated netlify preview.

Reply:
In the meantime, we can check the hard link in the live docs: 
https://docs.konghq.com/0.12.x/configuration/#nginx-section